xds/internal/client/client_test.go

```diff
@@ -58,6 +59,20 @@ const (
 	defaultTestShortTimeout       = 10 * time.Millisecond // For events expected to *not* happen.
 )
 
+var (
+	cmpOpts = cmp.Options{
+		cmpopts.EquateEmpty(),
+		cmp.Comparer(func(a, b time.Time) bool { return true }),
+		cmp.Comparer(func(x, y error) bool {
+			if x == nil || y == nil {
```

Comment:
Does `errors.Is()` work here instead?

Reply:
`errors.Is()` doesn't work for errors with the same error message I think. It compares the pointer.

And we still support 1.11.